test/test-npm-install.js

```diff
@@ -68,6 +68,25 @@ test('npm-install: no package.json', function (t) {
   });
 });
 
+test('npm-install: timout', function (t) {
+  var context = {
+    emit: function() {},
+    path: sandbox,
+    module: {
+      name: 'omg-i-pass'
+    },
+    meta: {},
+    options: {
+      npmLevel: 'silly',
+      timeoutLength: 100
+    }
+  };
+  npmInstall(context, function (err) {
+    t.equals(err.message, 'Install Failed');
```

Comment:
Might want to do `t.equals(err && err.message, 'Install Failed');` here otherwise the failure case is can be really annoying (exception thrown because `undefined has no property 'message'` vs. intelligible test failure message).